.github/workflows/lint.yml

```diff
@@ -2,11 +2,9 @@ name: Lint
 
 on:
   push:
-    branches-ignore:
-      - 'artifacts'
+    branches:
+      - main
   pull_request:
-    branches-ignore:
-      - 'artifacts'
```

Comment:
There's no need to exclude `artifacts` for this trigger, as pull requests are never made against it.